internal/services/storage/storage_account_network_rules_resource_test.go

```diff
@@ -426,13 +423,20 @@ resource "azurerm_storage_account_network_rules" "test" {
   ip_rules                   = []
   virtual_network_subnet_ids = []
 }
-`, StorageAccountResource{}.networkRulesPrivateEndpointTemplate(data), data.RandomString)
+`, StorageAccountResource{}.networkRulesTemplate(data), data.RandomString)
 }
 
 func (r StorageAccountNetworkRulesResource) privateLinkAccess(data acceptance.TestData) string {
 	return fmt.Sprintf(`
 %s
 
+resource "azurerm_search_service" "test" {
+  name                = "acctestsearchservice%d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  sku                 = "basic"
```

Comment:
sanity checking: does the standard sku of search support private link? else this'd break in the future

Reply:
@tombuildsstuff Yes, I've tried the `standard` sku, which works fine. I believe the sku doesn't matter as otherwise it should be mentioned in https://learn.microsoft.com/en-us/azure/storage/common/storage-network-security?tabs=azure-portal#trusted-access-based-on-a-managed-identity?